src/app/tab-preferences/notification-base.ts

```diff
@@ -86,30 +95,31 @@ export class NotificationBase implements OnInit {
   }
 
   remoteNotifyLoadedOnce = false
-  public async loadNotifyToggles() {
+  public async loadNotifyToggles(isNotifyClientUpdatesEnabled: boolean) {
```

Comment:
My suggestion would be to move the `isNotifyClientUpdatesEnabled()` method from the sync.service to notification-base. Sync service has the context of notification-base so no issue there, you don't have to pass additional data here, and logically the method should be in notification-base to begin with since it should be aware of its own notification settings